packages/lib/constants.ts

```diff
@@ -152,7 +152,8 @@ export const BOOKER_NUMBER_OF_DAYS_TO_LOAD = parseInt(
 export const CLOUDFLARE_SITE_ID = process.env.NEXT_PUBLIC_CLOUDFLARE_SITEKEY;
 export const MINIMUM_NUMBER_OF_ORG_SEATS = 30;
 export const ORG_SELF_SERVE_ENABLED = process.env.NEXT_PUBLIC_ORG_SELF_SERVE_ENABLED === "1";
-export const ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE = 0;
+export const ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE =
+  process.env.NEXT_PUBLIC_ORG_MINIMUM_PUBLISHED_TEAMS_SELF_SERVE || 2;
```

Comment:
Introduced a new env variable here, just to give flexibility to change this value, but the default value is 2, i.e someone needs to create **two or more** teams **to view that option** and **also to be able to create a org**.